cmd/copy.go

```diff
@@ -62,9 +64,21 @@ func Copy(cmd *cobra.Command, args []string) error {
 	}
 
 	ui.Infof(fmt.Sprintf("🔄 Copying %s => %s", scpArgs[0], scpArgs[1]))
-	err := copySourceSCP(scpArgs[0], scpArgs[1], publicKeyPath)
+
+	// If the first argument of the SCP args points towards a logical directory assume local => remote
+	pathInfo, _ := os.Stat(scpArgs[0])
```

Comment:
This shouldn't ignore the error. The dereference below will cause a panic if the file doesn't exist. Also, we probably shouldn't assume the direction. We know that the remote path will always be prefixed by `sess:`. Let's use that info  to determine the direction.

Reply:
That's basically doing the same thing no? We are using the knowledge that the remote path is _not_ prefixed by sess, but yeah I'll clean this up, it's a bit hacky